ios/AppDelegate.m

```diff
@@ -52,6 +54,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   [[ExposureManager shared] broadcastCurrentEnabledStatus];
 
   [RNSplashScreen showSplash:@"LaunchScreen" inRootView:rootView];
+  //Crashlytics test
+  //assert(NO);
```

Comment:
How do we feel about removing this commented code? If this is something we will need in the future we can circle back to the implementation or the docs setting up the library?